apps/ledger-live-desktop/scripts/ci-lint.mjs

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env zx
+import { basename } from 'path'
+import stylish from "../../../node_modules/eslint/lib/cli-engine/formatters/stylish.js"
+
+const usage = () => {
+  console.log(`Usage: ${basename(__filename)} [-h] [-p <port>] `);
+  process.exit(1)
+}
+
+let port;
+
+for (const arg in argv) {
+  switch (arg) {
+    case 'h':
+      usage();
+      break;
+    case 'p':
+      port = argv[arg];
+      break
+    case '_':
+      break;
+    default:
+      usage();
+      break;
+  }
+}
+
+const lint = async () => {
+  cd('../../')
+
+  if (typeof port === 'string') {
+    await $`pnpm lint \\
+      --filter="ledger-live-desktop" \\
+      --api="http://127.0.0.1:${port}" \\
+      --token="yolo" \\
+      --team="foo" \\
+      -- --format="json" \\
+      -o="lint.json"`;
+  } else {
+    await $`pnpm lint \\
+      --filter="ledger-live-desktop" \\
+      --token="yolo" \\
```

Comment:
sorry I have added this as a parameter, we need to pass the server token (yolo) as argument